src/environment.js

```diff
@@ -37,6 +37,8 @@ class Environment {
       return 'heroku';
     } else if (this._env.GITLAB_CI == 'true') {
       return 'gitlab';
+    } else if (this._env.TF_BUILD == 'True') {
```

Comment:
Is there a reason these are `==` rather than `===`?

Reply:
No, I don't think so.  I won't change them all in this PR, but perhaps they should be updated in a subsequent PR.